modules/integration/tests-common/admin-clients/src/main/java/org/wso2/mb/integration/common/clients/AndesJMSPublisher.java

```diff
@@ -109,13 +109,16 @@ public AndesJMSPublisher(AndesJMSPublisherClientConfiguration config, boolean cr
                     .lookup(AndesClientConstants.CF_NAME);
             connection = connFactory.createConnection();
             connection.start();
-            this.session = connection.createSession(false, Session.AUTO_ACKNOWLEDGE);
+            if(config.isTransactionalSession()) {
+                this.session = connection.createSession(true, 0);
```

Comment:
Creating a transacted session for publisher invalidate the value provided for acknowledgeMode. Hence used 0.